src/scripts/main.js

```diff
@@ -1,5 +1,32 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const body = document.body;
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+document.addEventListener('click', (e) => {
+  const positionZero = 0 + 'px';
+
+  if (e.target === body) {
+    return;
+  }
+
+  spider.style.top = e.clientY - wall.offsetTop - 35 + 'px';
```

Comment:
you have some hardcoded values (`35` and `15`). please ruse proper elements and get their width or height